### PR TITLE
Update manifest.json

### PR DESCRIPTION
**Pull request by ViktarStarastsenka**
Changes to the summary for the search and query tutorial based on https://github.com/RedisInsight/Tutorials/pull/28#discussion_r1470100738

### CODE REVIEW

File: src/manifest.json [hunk 1]
```diff
@@ -291,7 +291,7 @@
           "type": "internal-link",
           "id": "sq-intro",
           "label": "Introduction",
-          "summary": "Try real-time searching and perform complex structured queries without compromising the database performance.",
+          "summary": "Enable Redis as your main database performing structured queries without compromising performance and simplicity.",
```
Comment (dwdougherty) [inline src/manifest.json:294]:
```suggestion
          "summary": "Use Redis as your main database to perform structured queries without compromising performance and simplicity.",
```

File: src/manifest.json [hunk 2]
```diff
@@ -300,6 +300,7 @@
           "type": "internal-link",
           "id": "sq-exact-match",
           "label": "Exact match",
+          "summary": "Matching values in your datasets' attributes for secondary index use cases.",
```
Comment (dwdougherty) [inline src/manifest.json:303]:
```suggestion
          "summary": "Match values in your datasets' attributes for secondary index use cases.",
```

File: src/manifest.json [hunk 3]
```diff
@@ -308,6 +309,7 @@
           "type": "internal-link",
           "id": "sq-full-text",
           "label": "Full-text search",
+          "summary": "Leveraging advanced text search capabilities to retrieve the most relevant datasets.",
```
Comment (dwdougherty) [inline src/manifest.json:312]:
```suggestion
          "summary": "Leverage advanced text search capabilities to retrieve the most relevant datasets.",
```

File: src/manifest.json [hunk 4]
```diff
@@ -316,6 +318,7 @@
           "type": "internal-link",
           "id": "sq-range",
           "label": "Range queries",
+         "summary": "Query on numeric fields that returns the values that are in between a given start and end value, such as prices, dates.",
```
Comment (dwdougherty) [inline src/manifest.json:321]:
```suggestion
         "summary": "Perform queries on numeric fields to return the values that are in between a given start and end value, such as prices and dates.",
```

File: src/manifest.json [hunk 5]
```diff
@@ -324,6 +327,7 @@
           "type": "internal-link",
           "id": "sq-geospatial",
           "label": "Geospatial queries",
+          "summary": "Simple point-based queries and beyond including complex geometries for more detailed querying of spatial data.",
```
Comment (dwdougherty) [inline src/manifest.json:330]:
```suggestion
          "summary": "Perform geometric queries on spatial data.",
```

**Review by dwdougherty — CHANGES_REQUESTED** [review]
Just a few changes.

File: src/manifest.json (hunk 4 above)
Comment (ViktarStarastsenka) [reply src/manifest.json:321]: need to cut 10 characters to fit the screen
what do you think about:
`Perform queries on numeric fields to return the values that are between a start and end value, such as prices and dates.`

**Pull request closed without merging**